src/app/core/api/model/breadcrumb.ts

```diff
@@ -0,0 +1,4 @@
+export interface Breadcrumb {
+  label: string;
+  url: string;
+}
```

Comment:
Don't create anything in `src/app/core/api/` manually, this whole folder is auto-generated.